pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-osd/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, pidgin, xosd
+, autoconf, automake, libtool } :
+
+stdenv.mkDerivation rec {
+  name = "pidgin-osd-0.1.0";
+  src = fetchurl {
+    url = https://github.com/mbroemme/pidgin-osd/archive/pidgin-osd-0.1.0.tar.gz;
+    sha256 = "11hqfifhxa9gijbnp9kq85k37hvr36spdd79cj9bkkvw4kyrdp3j";
+  };
+
+  preConfigure = "./autogen.sh";
```

Comment:
It's better to use `autoreconfHook` here and remove `preConfigure` and the deps on `autoconf`, `automake`, and `libtool`. 

Just add `nativeBuildInputs = [ autoreconfHook ];`.